addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3885,21 +3881,10 @@ public Action:MakeNotBoss(Handle:timer, any:userid)
 	SetEntProp(client, Prop_Data, "m_iHealth", GetEntProp(GetPlayerResourceEntity(), Prop_Send, "m_iMaxHealth", _, client));
 	if(GetClientTeam(client)==BossTeam)
 	{
-		SetEntProp(client, Prop_Send, "m_lifeState", 2);
-		ChangeClientTeam(client, OtherTeam);
-		SetEntProp(client, Prop_Send, "m_lifeState", 0);
-		TF2_RespawnPlayer(client);
+		AssignTeam(client, TFTeam:OtherTeam, GetRandomInt(1,9));
```

Comment:
Pretty sure this will mess with TF2 trying to naturally reset the player's class to what it was before